apps/meteor/app/lib/server/lib/notifyUsersOnMessage.ts

```diff
@@ -59,52 +54,40 @@ export async function getMentions(message) {
 	};
 }
 
-const incGroupMentions = async (rid, roomType, excludeUserId, unreadCount) => {
+const incGroupMentions = async (rid: string, roomType: RoomType, excludeUserId: string, unreadCount: string): Promise<void> => {
```

Comment:
I think we can rename `unreadCount` to something that makes more sense
the 
```typescript
unreadCount: string
```
does not make a lot of sense, since counts should be integers, do you know if there is any reason for this, the type also looks weird.
If you look at the possible values of the settings `Unread_Count`, `Unread_Count_DM` and `Unread_Count_Omni` you can see, the possible values, they are `'all_messages', 'user_mentions_only', 'group_mentions_only', 'user_and_group_mentions_only'`
Maybe we can use this as a type

Reply:
also rid is of type IRoom['_id']
even tough it is a string we should follow this pattern
also
excludeUserId should be IUser['_id']

Reply:
Suggestions applied on https://github.com/RocketChat/Rocket.Chat/pull/31877/commits/08111f50d11d1514268c365ff11169454e5560c8. Additionally, I have decided to keep the type definition in the same file since it doesn't share that definition with any other file.